src/steps/organization/index.test.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+
+import { setupSigSciRecording } from '../../../test/recording';
+
+import { IntegrationConfig } from '../../config';
+import { fetchCorps } from './';
+import { integrationConfig } from '../../../test/config';
+
+describe('should collect corp data', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupSigSciRecording({
+      directory: __dirname,
+      name: 'buildCorp',
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('validate client integration and converters', async () => {
+    // Arrange
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
+
+    // Act
+    await fetchCorps(context);
+
+    // Assert
+    const organizations = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('Organization'),
+    );
```

Comment:
Just a tip - technically, `context.jobState.collectedEntities` should contain _only_ organizations. This is one of the benefits of these isolated tests.

```suggestion
    const organizations = context.jobState.collectedEntities;
```